cmake/testing.cmake

```diff
@@ -108,6 +108,8 @@ f3d_test(NAME TestPointCloud DATA pointsCloud.vtp ARGS -o --point-size=20)
 f3d_test(NAME TestPointCloudBar DATA pointsCloud.vtp ARGS -sob --point-size=20)
 f3d_test(NAME TestPointCloudUG DATA pointsCloud.vtu ARGS -o --point-size=20)
 f3d_test(NAME TestPointCloudVolume DATA bluntfin.vts ARGS -sob)
+f3d_test(NAME TestPointCloudDefaultScene DATA pointsCloud.vtp ARGS --point-size=20)
+f3d_test(NAME TestPointCloudFullScene DATA pointsCloud.gltf ARGS --point-size=20)
```

Comment:
tehcnically both test are the same code path, but I still think this is worth checking.